introductory/src/App.jsx

```diff
@@ -1,8 +1,45 @@
+import React from "react"
```

Comment:
Remove import react: In react 17  and above the JSX are transformed without React